doc/Client-Portal Protocol.md

```diff
@@ -90,7 +91,19 @@ This a format for client reconnects.
 
 **RequestName**: "relogin"<br>
 
-**RequestData**: The ReconnectionTicket object defined in 3.3.1 section.<br>
+**RequestData**: The ReconnectionTicket object defined in 3.3.1 section and PendingMessages:
+
+  object(ReloginResponse)::
+    {
+      ticket: string(Base64Encoded(object(ReconnectionTicket)))
+      messages: [
+        {
```

Comment:
If there is no messages, could it be `undefined` or must be an empty array?

Reply:
no "|| undefined" after "[ ]" means empty array only.